src/index.ts

```diff
@@ -28,6 +28,8 @@ function run(updateDetails?: IUpdateDetails) {
         --localonly, -l         Enable to not scan other directories than CWD for plugins
         --verbose, -v           Enable verbose logging
         --production, -P        Enable production mode (ignores test dependencies)
+        --workDir, -d           Directory of Cplace Project
```

Comment:
Do we really need this? I'd remove it again as everything else also only works when run directly inside the repo...